Packs/FireEyeETP/Integrations/FireEyeETP/FireEyeETP.py

```diff
@@ -478,6 +547,83 @@ def get_alerts_command():
     demisto.results(entry)
 
 
+def upload_yara_file_command(client, args):
+
+    entry_id = args.get('entryID')
+    policy_uuid = args.get('policy_uuid')
+    ruleset_uuid = args.get('ruleset_uuid')
+
+    file_obj = demisto.getFilePath(entry_id)
+    file_path = file_obj['path']
+
+    with open(file_path, "rb") as file:
+        data=file.read()
+        files = {
+            'file': ('new.yara', data)
+        }
+        response = client.upload_yara_file(policy_uuid, ruleset_uuid, files)
+        if response.status_code == 202:
+            return CommandResults(readable_output='Upload of Yara file succesfully.')
+        else :
+            return CommandResults(readable_output='Upload of Yara file failed.')
+
+
+def get_events_data_command(client, args):
+
+    message_id = args.get('message_id')
+
+    response = client.get_events_data(message_id)
+
+
+    result_output = dict()
+    result_output['Logs'] = response['data'][message_id]
+
+    for log in result_output['Logs']:
+        if log['action_on_msg'] == "MTA_RCPT_DELIVERED_OUTBOUND":
+            result_output['Delivered_msg'] = log['display_msg']
+            result_output['Delivered_status'] = "Delivered"
+            result_output['InternetMessageId'] = result_output['Delivered_msg'].split('<')[1].split('>')[0]
+        if log['action_on_msg'] == "MTA_RCPT_DELIVERY_PERM_FAILURE_OUTBOUND":
+            result_output['Delivered_msg'] = log['display_msg']
+            result_output['Delivered_status'] = "Failed"
+
+    command_results = CommandResults(outputs=result_output, readable_output=tableToMarkdown("Events", result_output, headers=["Logs", "Delivered_msg", "Delivered_status"]), outputs_prefix='FireEyeETP.Events')
```

Comment:
In CommandResults please try to use is_auto_json_transform=true so the logs will be displayed better- please attached a screen shot so we choose which version is better
![image](https://github.com/user-attachments/assets/1b2d2292-eaee-46da-8f12-7b9411054667)

Reply:
Done, i push changes now. This is the screen shot:
![image](https://github.com/user-attachments/assets/d4fcf380-dd56-4620-bef4-02755ce7f351)

Reply:
yes i think it is much better, thank you!